smoothscroll.js

```diff
@@ -101,13 +103,6 @@
         sy = w.scrollY || w.pageYOffset,
         startTime = now();
 
-    if (startX === undefined) {
-      startX = sx;
-      startY = sy;
-      endX = x;
-      endY = y;
-    }
-
```

Comment:
Just had a chance to test this version and moving start and end variables to local scope and erasing this prevents the script from crashing  if you call ```window.scroll``` while there's other smooth scrolling going on :sparkles: :sparkles: